.github/README.md

```diff
@@ -0,0 +1,371 @@
+
+# Create Release PR Workflow
+
+## Overview
+The `Create Release PR` GitHub Workflow automates the creation of a release pull request (PR) for Mojaloop. It updates Helm chart dependencies, generates a changelog, deploys a test environment on Kubernetes, and runs validation tests. This workflow is manually triggered via GitHub’s `workflow_dispatch` event, offering flexibility with customizable inputs for release naming, versioning, and deployment configuration.
+
+## Features
+- Creates a draft PR with updated Helm charts and release notes.
+- Deploys a test environment using Helm and Kubernetes.
+- Generates a changelog based on the last release tag.
+- Runs Testing Toolkit (TTK) tests to validate the deployment.
```

Comment:
Maybe number this bullet points and I think swap the last two steps (integrate with a repo to get config and then run tests)

Reply:
Done